### DISCUSSION TITLE
无法绑定账号

### DISCUSSION THREAD

**Discussion by scarlettf** [MoeNetwork/Tieba-Cloud-Sign#102] [疑问] [answered]
请问这种情况要怎样修复呢？我按照教程获取和扫码都是失败
![image](https://user-images.githubusercontent.com/56741347/123773575-56d51f80-d8ff-11eb-851c-ff090f116f57.png)
![image](https://user-images.githubusercontent.com/56741347/123773629-618fb480-d8ff-11eb-929a-0b7688ecb8ab.png)
![image](https://user-images.githubusercontent.com/56741347/123774014-a0be0580-d8ff-11eb-9788-b74e48bc2024.png)

**Comment by BANKA2017** [comment]
无法复现，能否把控制台Network的请求 `/ajax.php?mod=baiduid:qrlogin` 的返回内容发一下，敏感信息可以打码

**Reply by BANKA2017** [reply]
被csrf防住可能是nginx/apache等的配置不正确的问题导致的，可以尝试修改 [config.php](https://github.com/MoeNetwork/Tieba-Cloud-Sign/blob/master/config.php) 的 `ANTI_CSRF` 项为 `false` 临时解决

**Reply by BANKA2017** [reply]
经过重新检查，可能的原因是您在尝试绑定一个未设置用户名的百度账号（比如直接用手机号注册），目前云签不支持绑定此类帐号

**Reply by scarlettf** [reply]
非常感谢！本来有用户名的也只可以扫码，换成false后就可以手动添加BDUSS了

**Comment by kenvix** [comment, marked as answer]
最新版本已解决此问题
